elements/lisk-transaction-pool/src/types.ts

```diff
@@ -29,7 +29,7 @@ export interface TransactionObject {
 	signatures?: ReadonlyArray<string>;
 	readonly type: number;
 	verifiedOnce?: boolean;
-	feePriority?: bigint;
+	feePriority: bigint;
```

Comment:
i think this has to be optional because `feePriority` is not property of transaction. it is not assigned when inserting to the pool

Reply:
If we keep it optional and then assume that in response of `getProcessableTransactions` it will be available, that assumption is also wrong.

Reply:
yes i mean not read-only, is fine, but it still should be `optional`.
When inserting the transaction in to the pool the type has to match. However, the transaction class doesn't have this property, so it will not match and not build when framework become typescript

Reply:
I think we can assume that the `feePriority` coming from `getProcessableTransactions` will always present but when call the add on pool, the incoming tx, may or maynot have this value and for that, we need to keep it `optional` and non-readonly.